chromium_src/tools/grit/preprocess_if_expr.py

```diff
@@ -92,6 +117,13 @@ def parse_args(self, original_method, argv):
             manifest_data = json.load(open(manifest_path))
 
         for override_file in overrides:
+            if should_run_mangler(override_file):
+                upstream_file = override_file.replace('.mangle.html.ts',
+                                                      '.html.ts')
+                run_mangler(os.path.join(override_root_folder, override_file),
+                            os.path.join(out_folder, upstream_file))
+                continue
```

Comment:
do we expect both overrides to exist for a file (mangle and not mangle)? Does it make sense? If not, let's drop `.mangle` extension and remove `chromium_src` customization to support `.mangle` files. Instead to determine if a mangler should be run, scan the `.ts` file for `from 'lit_mangler'`.

Reply:
Happy to do that but the reason I introduced the `.mangle` extension was to make it obvious it's doing something different (i.e. at build time, rather than at runtime).

Reply:
That's fine, I agree it would be better to differentiate those files. Let's generalize the chromium_src overrides lookup a bit (see the comment in `util.js`).

Maybe make those files be more explicit with the extension? Say, `.lit_mangle.ts`?

Reply:
Yup, sounds good to me :smile: